package.json

```diff
@@ -16,6 +16,7 @@
     "react-scripts": "5.0.1",
     "react-test-renderer": "^18.1.0",
     "redux": "^4.2.0",
+    "redux-thunk": "^2.4.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
### You have not made changes to your **`README.md`**

- [ ] **Required:**  You have used the placeholder **Project Name** instead of replacing it with yours. Kindly add the title of your project and the description as well to make your project look professional.